src/app/app.module.ts

```diff
@@ -1,14 +1,18 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
-
-import { AppComponent } from './app.component';
+import { AppRoutingModule } from './app-routing.module';
+import { AppComponent } from '@app/app.component';
+import { Routes, RouterModule } from '@angular/router';
```

Comment:
Follow one codding standard (@app or ./app).
Predefined and custom components need to be serrated while importing